db/schema.rb

```diff
@@ -747,6 +747,8 @@
     t.jsonb "ui_settings", default: {}
     t.jsonb "custom_attributes", default: {}
     t.string "type"
+    t.boolean "email_signature_enabled", default: false, null: false
```

Comment:
Can we use a generic name here? We can use the same signature across channels.

Reply:
@pranavrajs Just `signature_enabled` will work?

Reply:
let us put it as message_signature and message_signature_enabled